src/pages/MyJobs/ClientJobs/ClientJob/ClientJob.tsx

```diff
@@ -7,34 +7,36 @@ import PageContainer, {
   TitleText,
 } from "pages/MyJobs/ClientJobs/ClientJobsStyles";
 import JobItem from "components/UI/JobItem/JobItem";
-
-const jobPost = {
-  title: "Freelancing website",
-  description: "Description",
-  publicationDate: "December 16",
-  payout: "$1500",
-  skills: ["JavaScript", "Python", "MySQL"],
-};
+import { useFetchDetailsQuery } from "services/jobs/jobs.api";
+import { useTranslation } from "react-i18next";
+import { format } from "date-fns";
 
 function ClientJob(): JSX.Element {
   const { jobId } = useParams();
-  // const { t } = useTranslation();
-
-  const { title, description, publicationDate, payout, skills } = jobPost;
+  const { t } = useTranslation();
+  const { data } = useFetchDetailsQuery({
+    id: Number(jobId),
+  });
 
   return (
     <PageContainer>
       <Header wd="70%">
         <TitleText font_sz="2.5em" pd="20px" pd_bottom="5%">
-          {title} {jobId}
+          {data?.title || t("NotFound.notFound")} {jobId}
           <TitleText font_sz="0.4em" pd="5px" pd_bottom="1%">
-            {publicationDate}
+            {data?.createdAt
+              ? format(new Date(data?.createdAt), "dd/MM/yyyy")
```

Comment:
move date format to consts. We already have date format const in inputPropsConsts.ts. Use it , or add new one